client/src/templates/Challenges/components/HelpModal.js

```diff
@@ -24,7 +24,7 @@ const propTypes = {
 };
 
 const RSA =
-  'https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck-coding/19514';
+  'https://forum.freecodecamp.org/t/how-to-get-help-when-you-are-stuck-coding/19514';
```

Comment:
```suggestion
  'https://forum.freecodecamp.org/t/19514';
```

Discourse is awesome, you do not need the slug in there, and it would still work.

Reply:
You can test this for example by following this link: https://www.freecodecamp.org/forum/t/19514

Reply:
That is a very cool feature!